BlueprintUICommonControls/Tests/Sources/AttributedLabelTests.swift

```diff
@@ -357,6 +357,70 @@ class AttributedLabelTests: XCTestCase {
         compareSnapshot(of: element)
     }
 
+    func test_linkAccessibility() {
+        let labelview = AttributedLabel.LabelView()
+
+        do {
+            // Test that link insertion happy path works
```

Comment:
Can we add a test for a link at the very end of the string too?

Reply:
done